include/session/config/profile_pic.h

```diff
@@ -6,14 +6,16 @@ extern "C" {
 
 #include <stddef.h>
 
+// Maximum length of the profile pic URL (not including the null terminator)
+extern const size_t PROFILE_PIC_MAX_URL_LENGTH;
+
 typedef struct user_profile_pic {
-    // Null-terminated C string containing the uploaded URL of the pic.  Will be NULL if there is no
-    // profile pic.
-    const char* url;
-    // The profile pic decryption key, in bytes.  This is a byte buffer of length `keylen`, *not* a
-    // null-terminated C string.  Will be NULL if there is no profile pic.
-    const unsigned char* key;
-    size_t keylen;
+    // Null-terminated C string containing the uploaded URL of the pic.  Will be length 0 if there
+    // is no profile pic.
+    char url[224];
```

Comment:
i feel like the size of `url` and the value of `PROFILE_PIC_MAX_URL_LENGTH` will very easily get out of sync here. if that happens that feels like a very easy to footcannon regression.

Reply:
It cannot get out of sync because there's a static_assert that will fail to compile if it does.